pom.xml

```diff
@@ -103,7 +103,6 @@
     <module>integration/spark-datasource</module>
     <module>integration/spark2</module>
     <module>integration/spark-common-test</module>
-    <module>integration/spark-carbon-common-test</module>
```

Comment:
Are all the testcases moved to spark-common-test module?

Reply:
yes